presto-docs/src/main/sphinx/optimizer/join-distribution-type.rst

```diff
@@ -0,0 +1,37 @@
+**********************************
+Repartitioned and Replicated Joins
+**********************************
+
+Background
+----------
+
+Presto can perform two types of distributed joins: repartitioned and replicated. In a repartitioned join, both inputs to a
```

Comment:
This text is rather wide which makes it harder to read. Please try to wrap text to be around ~80 characters.